app/routes/photos/components/GalleryPictureModal.js

```diff
@@ -6,6 +6,7 @@ import GalleryDetailsRow from './GalleryDetailsRow';
 import { Flex } from 'app/components/Layout';
 import { Content } from 'app/components/Content';
 import Icon from 'app/components/Icon';
+import throttle from 'lodash/throttle';
```

Comment:
We do have a babel plugin (https://github.com/webkom/lego-webapp/blob/master/package.json#L100) to automatically cherry pick lodash functions, so you can write `import { throttle } from 'lodash';`. Not important here ofc, but if you were to use more lodash functions you can avoid multiple lines.

Reply:
:+1:

Just a (bad?) habit from another project.

Reply:
Not bad without the plugin, tho 😉